tsl/src/bgw_policy/continuous_aggregate_api.c

```diff
@@ -81,15 +90,23 @@ get_interval_from_config(const Dimension *dim, const Jsonb *config, const char *
 }
 
 int64
-policy_refresh_cagg_get_refresh_start(const Dimension *dim, const Jsonb *config, bool *start_isnull)
+policy_refresh_cagg_get_refresh_start(const Dimension *dim, const Jsonb *config)
 {
-	return get_interval_from_config(dim, config, CONFIG_KEY_START_INTERVAL, start_isnull);
+	bool start_isnull;
+	int64 res = get_interval_from_config(dim, config, CONFIG_KEY_START_INTERVAL, &start_isnull);
+	if (start_isnull)
+		return PG_INT64_MIN;
```

Comment:
i think you should get the boundaries from the added utility functions instead of hardcoding

Reply:
done.